src/main.tf

```diff
@@ -9,7 +9,7 @@ resource "azurerm_log_analytics_workspace" "main" {
   name                = var.md_metadata.name_prefix
   resource_group_name = azurerm_resource_group.main.name
   location            = azurerm_resource_group.main.location
-  tags                = var.md_metadata.name_prefix
+  tags                = var.md_metadata.default_tags
 }
 
 resource "azurerm_kubernetes_cluster" "main" {
```

Comment:
Setting this would be similar to a network firewall, blocking all traffic but whitelisting a subset of IP addresses. As with Key Vault, we don't have a way of supporting this right now.